apps/nekohit/src/app/create-project/create-project.component.html

```diff
@@ -96,28 +96,23 @@
               </label>
               <div class="">
                 <p-inputNumber
-                  formControlName="securityStake"
+                  formControlName="securityStakePercent"
                   mode="decimal"
-                  [minFractionDigits]="
-                    tokenService.getTokenByHash(token.value).decimals
-                  "
-                  [maxFractionDigits]="
-                    tokenService.getTokenByHash(token.value).decimals
-                  "
-                  [suffix]="
-                    ' ' + tokenService.getTokenByHash(token.value).symbol
-                  "
-                  [max]="
-                    getTokenBalance(
-                      tokenService.getTokenByHash(token.value).symbol
-                    )
-                  "
-                  [min]="0"
+                  [minFractionDigits]="2"
```

Comment:
I suggest setting the digits to `0`. This field is `stakeRate100` in the contract, which should be an integer, and the minimal step is 1%

Reply:
okay this makes sense